openspec/changes/add-drift-alert-delivery/tasks.md

```
@@ -0,0 +1,29 @@
+## 1. Deliver-then-record drift alerts (#235)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)